pom.xml

```diff
@@ -1379,12 +1379,12 @@
       <dependency>
         <groupId>org.infinispan.protostream</groupId>
         <artifactId>protostream</artifactId>
-        <version>4.3.5.Final</version>
+        <version>4.4.0.Beta2</version>
```

Comment:
It's not clear to me why we need a beta version. The minimum would be to put a comment to explain it.

Reply:
Infinispan 12 requires version 4.4.0 of protostream, which is, as far as I know, only available as Alpha or Beta version.

Reply:
I will add a comment to explain it.

Reply:
I think I would prefer to wait until protostream 4.4 final is out. It's not like we were is a hurry to upgrade Infinispan.